app/posts/manage-teacher-training-applications/2020-06-24-end-of-cycle-exploratory-research.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Research - end of cycle (exploratory)
+description: Understand provider needs around ending a recruitment cycle. Also some usability testing of reasons for rejection, candidate guidance on the application view and sort by Reject by Default date
+date: 2020-06-18
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+## The context
+
+We know that there are specific activities providers undertake when a recruitment cycle is closing. This research was to understand provider needs for:
+* ending a recruitment cycle
+* handling "edge case" applications
+* planning a new cycle
+* data & reporting to DfE
+
+We also wanted to assess design changes for:
+* reasons for rejection
+* adding a way for providers to see the question guidance shown to candidates
+* sorting controls, adding the ability to sort by Reject By Default (RBD) date
+
+## Approach
+
+One to one, moderated, semi-structured sessions
+[Discussion guide](https://docs.google.com/document/d/1gbTev2fd9cAAkVgP6JDYoZXSL_1M7PUgA_DAZIJrwFU/)
+
+## Who we researched with
+
+Provider users, typically ITT administrators or ITT Directors from:
+* 7 SCITTs
+* 2 Schools Direct
+
+## Analysis
+
+[End of cycle Miro board analysis](https://miro.com/app/board/o9J_kqtdx_E=/)
+
+## New user needs identified
+
+[Provider user needs](https://trello.com/b/5m1pMTme/apply-provider-needs-e2e)
+
+New need categories:
+* End of cycle
+  * [Know upcoming dates](https://trello.com/c/w2C5r8Tp/177-i-need-to-know-the-upcoming-key-dates-from-dfe-in-advance-so-that-i-can-manage-my-time)
```

Comment:
Linking to individual Trello cards like this makes me wonder if we shouldn’t use the Trello API to use our boards to power the user needs we refer to on this site? 🧐

Reply:
![image](https://user-images.githubusercontent.com/470137/85858967-61bd1780-b7b4-11ea-9eb6-fa783b322f29.png)

Reply:
@paulrobertlloyd would that pull through the needs from the board even if it's not fully publicly viewable as @defong has illustrated above?

Reply:
Should be possible, yes. I’ll need to have a closer look at the API. Of course, this begs the question should the boards be publicly viewable anyway (assuming they contain non-identifiable information about participants)?

Reply:
@paulrobertlloyd yeah, there's no reason they shouldn't be publicly viewable by anyone who's reached them via the design history. I can't edit that though as Helen owns the board and that's a constraint of how the shared team boards were set up as best I can tell